### PR TITLE
[Feature] Adding tests cases to the project

### PR DESCRIPTION
**Pull request by victor-heliomar**
## Explanation

In order to improve the quality of the code and provide security about the code that will be constantly pushed to the project for me or contributors, I added some test cases to the project.

It covers the most of cases and helped me to add a better error handling to the project.

Also removed some files that was be added by accident here on folder /docs.

## Screenshots

image

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

File: cli.js [hunk 1]
```diff
@@ -3,10 +3,8 @@
 const { Command, Argument } = require('commander');
 const program = new Command();
 const supportedLanguages = require('./SUPPORTED-LANGUAGE.json');
-const { translate } = require('./utils/translate');
 const { configPath } = require('./utils/getConfigPath');
-const { getOrCreateTranslationFile } = require('./utils/getTranslationFile');
-const fs = require('fs');
+const { translateController } = require('./controllers/translateController');
```
Comment (Israel-Laguan) [inline cli.js:7]: I think moving these should have been another PR before, but no problem keeping the change

Comment (victor-heliomar) [reply cli.js:7]: Yea I agree that would be better on another PR. When I was creating the tests for this file I found with the problem than mocking functions for CLI commands didn't work 😅, so I had to separate this logic of the CLI interface to be able to test it individually

Comment (victor-heliomar) [reply cli.js:7]: I really appreciate you helping me with the review. Thanks!
